.github/workflows/build-and-validate.yaml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   build-validate:
     name: Build and validate
-    runs-on: ubuntu-18.04
```

Comment:
I see `latest` and `20.04` - which one should we use?

Reply:
Per: https://github.com/awslabs/fhir-works-on-aws-deployment/pull/843/files#r1160095634